ott/geometry/geometry.py

```diff
@@ -137,7 +137,8 @@ def cost_matrix(self) -> jnp.ndarray:
     if self._cost_matrix is None:
       # If no epsilon was passed on to the geometry, then assume it is one by
       # default.
-      cost = -jnp.log(self._kernel_matrix)
+      eps = jnp.finfo(self._kernel_matrix.dtype).tiny
```

Comment:
smallest_normal ?

Reply:
Doesn't seem to exist, both in `numpy` and in `jax`, despite the [docs](https://numpy.org/doc/stable/reference/generated/numpy.finfo.smallest_normal.html#numpy.finfo.smallest_normal) saying otherwise. Might need to open an issue to see why.